bootstrap/Platform_Management/Setup_Pluggable_Library.groovy

```diff
@@ -0,0 +1,39 @@
+// Constants
+def pluggableGitURL = "https://github.com/Accenture/adop-pluggable-scm"
+
+def platformManagementFolderName= "/Platform_Management"
+def platformManagementFolder = folder(platformManagementFolderName) { displayName('Platform Management') }
+
+// Jobs
+def setupPluggable = freeStyleJob(platformManagementFolderName + "/Setup_Pluggable_Library")
+ 
+// Setup setup_cartridge
+setupPluggable.with{
+    environmentVariables {
+        keepBuildVariables()
```

Comment:
I think this needs a "true" in the brackets: https://jenkinsci.github.io/job-dsl-plugin/#path/freeStyleJob-environmentVariables-keepBuildVariables